masterlist.yaml

```
@@ -525,5 +525,13 @@ plugins:
       - 'Graphic Herbalism TR Extra.ESP'
 
 #Any dynamic patches (not sure if multipatch is the  only one)
+  - name: 'Merged Objects.esp'
```

You could add the location for the mod.

```suggestion
  - name: 'Merged Objects.esp'
    url: [ 'https://www.nexusmods.com/morrowind/mods/46870/' ]
```